src/main/java/org/elasticsearch/common/geo/GeoPoint.java

```diff
@@ -119,15 +130,15 @@ public boolean equals(Object o) {
     public int hashCode() {
         int result;
         long temp;
-        temp = lat != +0.0d ? Double.doubleToLongBits(lat) : 0L;
+        temp = y != +0.0d ? Double.doubleToLongBits(y) : 0L;
         result = (int) (temp ^ (temp >>> 32));
-        temp = lon != +0.0d ? Double.doubleToLongBits(lon) : 0L;
+        temp = x != +0.0d ? Double.doubleToLongBits(x) : 0L;
         result = 31 * result + (int) (temp ^ (temp >>> 32));
         return result;
     }
 
     public String toString() {
-        return "[" + lat + ", " + lon + "]";
+        return "[" + y + ", " + x + "]";
```

Comment:
I realize this was already this way, but the format here (using brackets) looks like GeoJSON, so shouldn't it be `[x,y]`?